### PR TITLE
{cae}[foss/2023b] meshio v5.3.5

### PR DESCRIPTION
**Pull request by tanmoy1989**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
eb-mix.zao.se - Linux Ubuntu 24.04 LTS (Noble Numbat), x86_64, AMD Ryzen 9 3900X 12-Core Processor (zen2), Python 3.12.3
See https://gist.github.com/zao/04a26be0b372cd201bb3477ab74bc1c0 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/96fbcf9f056e2839997a89f45bfe7bf6 for a full test report.

File: easybuild/easyconfigs/m/meshio/meshio-5.3.5-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+name = 'meshio'
+version = '5.3.5'
+
+homepage = 'https://github.com/nschloe/meshio'
+description = "meshio is a tool for reading/writing various mesh formats representing unstructured meshes"
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+dependencies = [
+    ('Python', '3.11.5'),  # includes numpy
```
Comment (bedroge) [inline easybuild/easyconfigs/m/meshio/meshio-5.3.5-foss-2023b.eb:12]: This comment is actually not true anymore for recent Python versions, numpy is now part of SciPy-bundle. The other dependencies listed at https://github.com/nschloe/meshio/blob/v5.3.5/pyproject.toml#L40 are both part of Python-bundle-PyPI. Both bundles are dependencies of VTK, which is why this is probably installing without issues, but we should list them here explicitly.

Comment (bedroge) [reply easybuild/easyconfigs/m/meshio/meshio-5.3.5-foss-2023b.eb:12]:
```suggestion
    ('Python', '3.11.5'),
    ('Python-bundle-PyPI', '2023.10'),  # includes rich
    ('SciPy-bundle', '2023.11'),  # includes numpy
```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/324556b7bb9f7d9a4c7adad0664e1e7c for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @tanmoy1989!

**Pull request merged**